pkgs/tools/wayland/wl-mirror/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, meson
+, ninja
+, pkg-config
+, wayland
+, wayland-protocols
+, wayland-scanner
+, wlr-protocols
+, libGL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wl-mirror";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "Ferdi265";
+    repo = "wl-mirror";
+    rev = "v${version}";
+    sha256 = "1wjdjzj6h1q51yg70gdrq2yrgg6ihamcwhizxfrjq8955yy2y6ly";
+  };
+
+  postUnpack = ''
+    (
+      cd "$sourceRoot"
+      sed -i 's@WL_PROTOCOL_DIR "/usr@WL_PROTOCOL_DIR "${wayland-protocols}@g' CMakeLists.txt
+      sed -i 's@WLR_PROTOCOL_DIR "/usr@WLR_PROTOCOL_DIR "${wlr-protocols}@g' CMakeLists.txt
+    )
+  '';
```

```suggestion
  postPatch = ''
    sed -i 's@WL_PROTOCOL_DIR "/usr@WL_PROTOCOL_DIR "${wayland-protocols}@g' CMakeLists.txt
    sed -i 's@WLR_PROTOCOL_DIR "/usr@WLR_PROTOCOL_DIR "${wlr-protocols}@g' CMakeLists.txt
  '';
```

Also substituteInPlace warns when the pattern no longer matches.

---

Done, and switched to substituteInPlace